assignments/closure.js

```diff
@@ -1,21 +1,43 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
+function sayHello(name) {
```

Comment:
good job with this. you could also strip it down a bit more, the template literal does not require parens, and you can drop the curly braces from the say function.

```
function sayHello(name) {
  const text = `Hello ${name}`;
  const say = () => console.log(text);
  say();
}
```